l10n_es_aeat/models/aeat_report_tax_mapping.py

```diff
@@ -92,7 +92,7 @@ def _get_move_line_domain(self, codes, periods=None,
             periods = self.env['account.period'].search(
                 [('fiscalyear_id', '=', self.fiscalyear_id.id)])
         move_line_domain = [('company_id', 'child_of', self.company_id.id),
-                            ('tax_code_id', 'child_of', tax_codes.ids),
+                            ('tax_code_id', 'in', tax_codes.ids),
```

Comment:
Por qué cambias esto?

Reply:
Porque si no el parámetro 'include_children' que recibe la función '_get_move_line_domain' no tiene efecto. Si te fijas lo pases a True o a False al final acaba construyendo el domain con 'child_of' de tax_codes, por lo que va a devolver todos los hijos de los códigos pasados al método aunque se pase 'include_children' a False; en líneas 87 a 90 del código ya se obtienen los tax_codes a incluir si 'include_children' es True y por eso aquí debe ser un 'in' y no un 'child_of'.
Hasta ahora supongo que no ha sido necesario, pero ahora para la casilla 80 hay que calcular la base imponible de operaciones realizadas en régimen general y para ello hay que sumar las bases al 21%, 10% y 4% y restar las modificaciones de bases por las rectificativas emitidas, pero el código de impuesto de éstas modificaciones (MBYCRBI) es 'padre' del código de modificaciones de bases repercutido por ISP (MBYCRBIISN), y si no se suma sin hijos (include_children=False) va a restar de la base de operaciones emitidas las modificaciones debidas a las facturas rectificativas de proveedor con ISP (que se meten en el código MBYCRBIISN con signo negativo).